packages/devextreme/js/__internal/ui/map/m_provider.google_static.ts

```diff
@@ -2,17 +2,17 @@
 import Color from '@js/color';
 import { name as clickEventName } from '@js/common/core/events/click';
 import eventsEngine from '@js/common/core/events/core/events_engine';
-import { each } from '@js/core/utils/iterator';
 import { getHeight, getWidth } from '@js/core/utils/size';
 
 import Provider from './m_provider';
+import type { LocationOption, MarkerOptions, RouteOptions } from './m_provider.dynamic';
 
 let GOOGLE_STATIC_URL = 'https://maps.google.com/maps/api/staticmap?';
 
 class GoogleStaticProvider extends Provider {
-  _locationToString(location) {
+  _locationToString(location: LocationOption): string {
     const latLng = this._getLatLng(location);
-    return latLng ? `${latLng.lat},${latLng.lng}` : location.toString().replace(/ /g, '+');
```

Comment:
hm, is omit of `.toString` indended here?

Reply:
well i didn't find a scenario when it's not string but let revert just in case